client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigationInfoBar.tsx

```diff
@@ -1,15 +1,90 @@
 import React from 'react';
+import { Button } from '@material-ui/core';
+
+import axios from 'Utils/axios';
+import { InvestigationInfo } from 'models/InvestigationInfo';
 
 import InvestigatedPersonInfo from './InvestigatedPersonInfo/InvestigatedPersonInfo';
 import InvestigationMetadata from './InvestigationMetadata/InvestigationMetadata';
 
-const InvestigationInfoBar = () => {
+const defaultInvestigationStaticInfo = {
+    startTime: new Date(),
+    lastUpdateTime: new Date(),
+    investigatingUnit: '',
+    userByCreator: {
+        personByPersonId: {
+            firstName: '',
+            lastName: '',
+            phoneNumber: '',
+            identificationType: '',
+            identificationNumber: '',
+            additionalPhoneNumber: '',
+            gender: '',
+            birthDate: new Date(),
+        }
+    },
+    userByLastUpdator: {
+        personByPersonId: {
+            firstName: '',
+            lastName: '',
+            phoneNumber: '',
+            identificationType: '',
+            identificationNumber: '',
+            additionalPhoneNumber: '',
+            gender: '',
+            birthDate: new Date(),
+        }
+    },
+    investigatedPatientByInvestigatedPatientId: {
+        personByPersonId: {
+            firstName: '',
+            lastName: '',
+            phoneNumber: '',
+            identificationType: '',
+            identificationNumber: '',
+            additionalPhoneNumber: '',
+            gender: '',
+            birthDate: new Date(),
+        },
+        isDeceased: false
+    },
+    coronaTestDate: new Date()
+}
```

Comment:
i dont really know what you should do here but i hate seeing such a long const defined in a component.
maybe define it in the investigationInfo model class and export it there?

Reply:
i will think about it